src/org/linphone/setup/GenericLoginFragment.java

```diff
@@ -181,7 +279,24 @@ else if(id == R.id.ab_back)
 			getActivity().onBackPressed();
 	}
 
-	
+	private class ProviderNetworkOperation extends AsyncTask<Void, Void, Void> {
+		@Override
+		protected Void doInBackground(Void... params) {
+			String query = "_rueconfig._tcp.aceconnect.vatrp.net";
+			srvLookup(query);
```

Comment:
This is showing successful SRV lookup example and should have documentation comment explaining it.